### DISCUSSION TITLE
Aiuto prima configurazione

### DISCUSSION THREAD

**Discussion by ergal88** [indomus/forum#1207] [HUB: Home Assistant]
Ciao a tutti,

come giusto che sia, visto che sono alle prime armi e per evitare di tartassare il gruppo telegram, ho pensato di aprire qui un thread dove chiedervi consiglio sulle configurazioni e sui primi passi da seguire.

Col vostro aiuto sono già riuscito ad installare e configurare MQTT Broker e Zigbee2mqtt in modalità docker compose sul mio PI4.

Al momento però sto sbattendo la testa con la ConBee II (aggiornata a fw 0x26720700) che sembra non rilevare nessun dispositivo

Su Windows + deConz rileva praticamente in modo immediato i vari device (test fatti con telecomando Ikea base e una lampadina 806lm sempre Ikea)

un aiutino?? 

grazie in anticipo a tutti

**Comment by Scialla** [comment]
La prolunga! :D :D :D

**Reply by ergal88** [reply]
maledetta prolunga..... :D
